clusterloader2/testing/load/config.yaml

```diff
@@ -339,6 +371,148 @@ steps:
       {{end}}
 {{end}}
 
+# BEGIN scheduler throughput
```

Comment:
@mm4tt @mborsz - I started wondering about this.
Given we look at max from the samples, Actually wouldn't it make sense to start it at the beginning and gather at the end? [Not necessary in this PR, but it would allow us to join this with the SLO measurement once we have that done.]

Reply:
I think we may want to do something like that in future, but before that we need to make sure that we create in test large enough deployments to make the scheduler be able to achieve full throughput. AFAIR currently we create max 250 pods per deployment, which should give ~2.5s of max scheduler throughput. I'm not sure if we are able to detect this with any measurement we have now (the current scheduler throughput uses 5s window and even if we switch to prometheus there, I expect similar window size).

If we start creating ~1k pods at the same time (to have > measurement windows to make it reliable), I think we can easily get rid of this part.

Reply:
What I had on my mind was basically leaving those additional deployments in this phase for now (since as you mentioned we need them), just moving scheduler throughput measurement (as it only looks at max 5s window).
So if there will be good window here, there will also be a good window across the whole test :)

But that's not for this particular PR - we can do that later.